internal/keycloakb/fields_validation.go

```diff
@@ -0,0 +1,44 @@
+package keycloakb
```

Comment:
I do not find any reference of this tool in the code. It is maybe a first version of the profile cache that is now in the keycloak-client